app/models/rdv.rb

```diff
@@ -152,7 +152,7 @@ def user_for_home_rdv
   end
 
   def overlapping_plages_ouvertures
-    return [] if starts_at.blank? || ends_at.blank? || lieu.blank? || past? || !valid?
+    return [] if starts_at.blank? || ends_at.blank? || lieu.blank? || past? || errors.present?
```

Comment:
:thinking: j'ai l'impression que ça ouvre des portes pour supprimer certains `form_object` :tada:

Reply:
En fait c’était pas directement lié, je sais pas si tu as vu le titre du commit pour ce changement:

> Fixup a side effect in overlapping_plages_ouvertures 
> Introduced in #1787 (5acd0a6). Calling valid? actually resets the validation. Since we’re now doing it _as part of the validation_, this clears any previous errors.

Avant, `overlapping_plages_ouvertures` n’était pas appelé au sein de `validate` (aka `valid?`) et la gestion des warnings et des erreurs étaient séparés. Maintenant, `overlapping_plages_ouvertures` est appelé par `validate`, donc si on rappelle `valid?` au milieu, en fait on redéclenche un cycle de validation, ce qui `.clear` les erreurs. (D’ailleurs ça m’étonne que ça ne fasse pas une récursion cheloue; peut-être que c’est le cas.)